okhttp/src/main/java/okhttp3/internal/concurrent/TaskRunner.kt

```diff
@@ -113,8 +113,8 @@ class TaskRunner(
   }
 
   private fun cancelAll() {
-    for (queue in activeQueues) {
-      queue.cancelAll()
+    for (i in activeQueues.size - 1 downTo 0) {
```

Comment:
```suggestion
    for (queue in activeQueues.asReversed()) {
```

Reply:
Tragically this allocates!

Reply:
It’s a view IIUC

Reply:
Creative the view itself is the allocation. I'm really trying to limit allocations here!

Reply:
It is a view but also its iterator.